README.md

```diff
@@ -8,9 +8,9 @@
 
 Animate a counter to `10`, from whatever value it currently is. For more advanced usage, see below.
 
-```sh
-npm install react-motion
-```
+Package managers:
```

Comment:
Just to be super newb-proof, can you do it this way:

Npm:
`npm install react-motion`

Bower:
`bower install react-motion`